src/common/com/intellij/plugins/haxe/ide/inspections/HaxePreprocessorInspection.java

```diff
@@ -82,7 +74,7 @@ public String getShortName() {
     List<ASTNode> nodes = new ArrayList<ASTNode>();
 
     ASTNode leaf = firstLeaf;
-    do {
+    while (leaf != null) {
```

Comment:
do while loop is resulting in NPE at line 78 (leaf.getElementType()) when firstLeaf is null.